platforms/src/target/env.rs

```diff
@@ -35,6 +35,12 @@ pub enum Env {
     /// `nto71`
     Nto71,
 
+    /// `nto71_iosock`
+    Nto71_iosock,
```

Comment:
I guess this should be:

```suggestion
    Nto71Iosock,
```

There's some code for doing various enum variant mapping where we either need to special case this, or change it to be the default way that snake case is transformed into camel case.